### ISSUE TITLE
Fail on task: Update Semaphore local admin account

### ISSUE THREAD

**Issue by lkthomas** [Thulium-Drake/ansible-role-ansible_semaphore#5]
TASK [thulium_drake.ansible_semaphore : Update Semaphore local admin account]

It's interesting that the code have a block and rescue section, is it possible to rework the logic behind? Would it be better to check if the account exists before doing update or creating an local admin account?

**Comment by Thulium-Drake** [comment]
Well, it seems I missed this ;-)

I don't actively use Semaphore anymore, but IIRC there's no CLI command to actually list all users in the installation (unless that has changed in the meantime)... That's why I resorted to "when updating fails, that's prolly because the user doesn't exist"

**Comment by bbaassssiiee** [comment]
```
# semaphore user list --config /etc/semaphore/semaphore.conf
Loading config
Validating config
admin
```

**Comment by Thulium-Drake** [comment]
Nice! I'll rework the tasks and make an update :-)

**Comment by Thulium-Drake** [comment]
Fixed in 0.0.7
